diff -Naur freepbx.ori/installlib/installcommand.class.php freepbx/installlib/installcommand.class.php
--- freepbx.ori/installlib/installcommand.class.php	2016-07-14 05:35:26.000000000 +0200
+++ freepbx/installlib/installcommand.class.php	2016-07-29 10:33:09.645165225 +0200
@@ -132,8 +132,8 @@
 	protected function execute(InputInterface $input, OutputInterface $output) {
 		global $amp_conf; /* This makes pandas sad. :( */
 
-		if (version_compare(PHP_VERSION, '5.6.19', '<')) {
-			$output->writeln("<error>FreePBX Requires PHP Version 5.6.19 or Higher, you have: ".PHP_VERSION."</error>");
+		if (version_compare(PHP_VERSION, '5.6.5', '<')) {
+			$output->writeln("<error>FreePBX Requires PHP Version 5.6.5 or Higher, you have: ".PHP_VERSION."</error>");
 			return false;
 		}
 
diff -Naur freepbx.ori/install.php freepbx/install.php
--- freepbx.ori/install.php	2016-07-14 05:35:26.000000000 +0200
+++ freepbx/install.php	2016-07-29 10:33:58.820845869 +0200
@@ -1,8 +1,8 @@
 <?php
 if (!defined('FREEPBX_IS_AUTH')) { die('No direct script access allowed'); }
 
-if (version_compare(PHP_VERSION, '5.6.19', '<')) {
-	out(sprintf(_("FreePBX Requires PHP Version 5.6.19 or Higher, you have: %s"),PHP_VERSION));
+if (version_compare(PHP_VERSION, '5.6.5', '<')) {
+	out(sprintf(_("FreePBX Requires PHP Version 5.6.5 or Higher, you have: %s"),PHP_VERSION));
 	return false;
 }
 
