diff -Naur freepbx/amp_conf/htdocs/admin/libraries/utility.functions.php freepbx.new/amp_conf/htdocs/admin/libraries/utility.functions.php
--- freepbx/amp_conf/htdocs/admin/libraries/utility.functions.php	2016-07-22 18:16:45.000000000 +0000
+++ freepbx.new/amp_conf/htdocs/admin/libraries/utility.functions.php	2016-08-02 07:34:37.437635345 +0000
@@ -1191,7 +1191,7 @@

 	if ($user !== false && $user != '') {
 		$sed_arg = escapeshellarg('s/\s*\[general\].*$/TEMPCONTEXT/;s/\[.*\]/\[' . $amp_conf['AMPMGRUSER'] . '\]/;s/^TEMPCONTEXT$/\[general\]/');
-		exec("sed -i.bak $sed_arg $conf_file", $output, $ret);
+		exec("/usr/bin/sed -i.bak $sed_arg $conf_file", $output, $ret);
 		if ($ret) {
 			freepbx_log(FPBX_LOG_ERROR,sprintf(_("Failed changing AMI user to [%s], internal failure details follow:"),$amp_conf['AMPMGRUSER']));
 			foreach ($output as $line) {
@@ -1203,10 +1203,10 @@
 	if ($pass !== false && $pass != '') {
 		unset($output);
 		$sed_arg = escapeshellarg('s/secret\s*=.*$/secret = ' . $amp_conf['AMPMGRPASS'] . '/');
-		exec("sed -i.bak $sed_arg $conf_file", $output2, $ret2);
+		exec("/usr/bin/sed -i.bak $sed_arg $conf_file", $output2, $ret2);
 		if ($ret2) {
 			freepbx_log(FPBX_LOG_ERROR,sprintf(_("Failed changing AMI password to [%s], internal failure details follow:"),$amp_conf['AMPMGRPASS']));
-			foreach ($output as $line) {
+			foreach ($output2 as $line) {
 				freepbx_log(FPBX_LOG_ERROR,sprintf(_("AMI failure details:"),$line));
 			}
 		}
@@ -1228,10 +1228,10 @@
 	unset($output);
 	if ($writetimeout) {
 		$sed_arg = escapeshellarg('s/writetimeout\s*=.*$/writetimeout = ' . $amp_conf['ASTMGRWRITETIMEOUT'] . '/');
-		exec("sed -i.bak $sed_arg $conf_file", $output3, $ret3);
+		exec("/usr/bin/sed -i.bak $sed_arg $conf_file", $output3, $ret3);
 		if ($ret3) {
 			freepbx_log(FPBX_LOG_ERROR,sprintf(_("Failed changing AMI writetimout to [%s], internal failure details follow:"),$amp_conf['ASTMGRWRITETIMEOUT']));
-			foreach ($output as $line) {
+			foreach ($output3 as $line) {
 				freepbx_log(FPBX_LOG_ERROR,sprintf(_("AMI failure details:"),$line));
 			}
 		}
